samples/unity-of-bugs/Assets/Scripts/BugFarm.cs

```diff
@@ -112,8 +113,9 @@ public void CrashNative()
     [MethodImpl(MethodImplOptions.NoInlining)]
     private void MethodB() => MethodA();
 
-    [DllImport("__Internal")]
-    private static extern void crash();
+    // Doesn't work on CI
+    //[DllImport("__Internal")]
+    //private static extern void crash();
```

Comment:
This errors out with:

```
Building Player failed

Exception: Unity.IL2CPP.Building.BuilderFailedException: C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.28.29333\bin\HostX64\x64\link.exe /out:"D:\a\sentry-unity\sentry-unity\samples\unity-of-bugs\Library\il2cpp_cache\linkresult_E7BDA3B41EA66A44456A6A4A71B00E57\GameAssembly.dll" /ignore:4206 /DEBUG /INCREMENTAL:NO /LARGEADDRESSAWARE /NXCOMPAT /DYNAMICBASE /NOLOGO /TLBID:1 /OPT:REF /OPT:ICF /HIGHENTROPYVA /DLL /IGNORE:4104 /NODEFAULTLIB:uuid.lib "kernel32.lib" "user32.lib" "advapi32.lib" "ole32.lib" "oleaut32.lib" "Shell32.lib" "Crypt32.lib" "psapi.lib" "version.lib" "MsWSock.lib" "ws2_32.lib" "Iphlpapi.lib" "Dbghelp.lib" "C:\Program Files\Unity\Editor\Data\PlaybackEngines\WindowsStandaloneSupport\Variations\win64_nondevelopment_il2cpp\baselib.dll.lib" /LIBPATH:"C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.28.29333\lib\x64" /LIBPATH:"C:\Program Files (x86)\Windows Kits\10\Lib\10.0.19041.0\um\x64" /LIBPATH:"C:\Program Files (x86)\Windows Kits\10\Lib\10.0.19041.0\ucrt\x64" /SUBSYSTEM:WINDOWS @"C:\Users\runneradmin\AppData\Local\Temp\tmpB224.tmp"

Creating library D:\a\sentry-unity\sentry-unity\samples\unity-of-bugs\Library\il2cpp_cache\linkresult_E7BDA3B41EA66A44456A6A4A71B00E57\GameAssembly.lib and object D:\a\sentry-unity\sentry-unity\samples\unity-of-bugs\Library\il2cpp_cache\linkresult_E7BDA3B41EA66A44456A6A4A71B00E57\GameAssembly.exp
26EB5DAB3CB6C7A85AFBF73691204B7F.obj : error LNK2019: unresolved external symbol crash referenced in function BugFarm_CrashNative_mB5FDCD440E3142A817FAE6AA5E877094C65E23F3
D:\a\sentry-unity\sentry-unity\samples\unity-of-bugs\Library\il2cpp_cache\linkresult_E7BDA3B41EA66A44456A6A4A71B00E57\GameAssembly.dll : fatal error LNK1120: 1 unresolved externals

   at Unity.IL2CPP.Building.CppProgramBuilder.PostprocessObjectFiles(HashSet`1 objectFiles, CppToolChainContext toolChainContext)
   at Unity.IL2CPP.Building.CppProgramBuilder.Build(IBuildStatistics& statistics)
   at il2cpp.Compilation.CompilationDriver.Run(RuntimePlatform platform, BuildingOptions buildingOptions, ReadOnlyCollection`1 matchedAssemblyMethodSourceFiles)
   at il2cpp.Program.DoRun(String[] args, RuntimePlatform platform, BuildingOptions buildingOptions, Boolean throwExceptions)

Failed to build player.
```

Reply:
My guess: The native method call `crash` returning void isn't getting compiled

So il2cpp validates that and fails the build.

Reply:
Yeah, but where is the source code for that native method?

Reply:
https://github.com/bruno-garcia/unity-of-bugs/blob/d7a5b96bcd41ff1195b94c700c21fa63149fb660/Assets/Plugins/Android/NativePlugin.c

Reply:
Requires IL2CPP though to compile ^